mindsdb/integrations/handlers/plaid_handler/README.md

```diff
@@ -0,0 +1,88 @@
+# Automate Financial Data Processing with MindsDB and Plaid
+
+Are you tired of manually processing financial data? With MindsDB and Plaid, you can easily automate this tedious task.
+
+First, connect your Plaid account by following these steps. Once you have your client ID, secret, and public key, create a database in MindsDB:
+
+```sql
+CREATE DATABASE my_plaid 
+WITH 
+    ENGINE = 'plaid',
+    PARAMETERS = {
+      "client_id": "YOUR_CLIENT_ID",
+      "secret": "YOUR_SECRET",
+      "access_token": "YOUR_PUBLIC_KEY",
+      "plaid_env": "ENV"
+    };
+```
+
```

Comment:
Can we please add hera a link to Plaid docs for generating secret and access token?